public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This file returns the favicon and other elements to be rendered in index.html which we won't be using. Remove this file as well